### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by vdunaev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 #### v2.3.0
 
 - Bump to math.gl@3.3.0 (#968)
+- Gltf up axis support (#965)
```
Comment (ibgreen) [inline CHANGELOG.md:8]: This is a new release `### 2.3.1`

Comment (ibgreen) [reply CHANGELOG.md:8]: Not sure where the attributes loader came from, but it is OK.

**Review by ibgreen — APPROVED** [review]
Make a new section for the new release

File: CHANGELOG.md (hunk 1 above)
Comment (vdunaev) [reply CHANGELOG.md:7]: Changed to 2.3.1

**Pull request merged**